src/screens/CreatePost/index.js

```diff
@@ -610,6 +622,7 @@ const CreatePost = () => {
 
   const isPollButtonDisabled = () => getReducedPoll().length < 2;
   const isButtonDisabled = () => {
+    if (loadingPost) return true;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line checks if `loadingPost` is true and returns true if it is. This is a good practice as it disables the button while the post is being created, preventing multiple submissions.
```diff
+    if (loadingPost) return true;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->